charts/turing/values.yaml

```diff
@@ -1,6 +1,10 @@
 global:
   protocol: http
 
+rendered:
+  releasedVersion: 1.11.0
+  ensemblerTag: v0.0.0-build.321-78ca7b3
```

Comment:
Should we also unset the `deployment.image.tag` field here? i.e. make it `""`

Reply:
Fixed